activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   For PostgreSQL >= 9.4 use `pgcrypto`'s `gen_random_uuid()` instead of
+    `uuid-ossp`'s UUID generation function.
+
+    *Yaw Boakye*
```

Comment:
@kamipo I disagree. Here's why.

I don't know whether I'll call https://github.com/rails/rails/pull/21327 original. I discovered it when I wanted to submit the improvements above. Some things were wrong and a couple of incorrect statements. So instead of sending another PR I rather left comments for the author's perusal. (See my comments.) I wasn't going to take it on until @senny extended the opportunity to me. And then what started as minor changes turned to a big overhaul, and a constant rebase to keep it relevant. If I still can't credit myself for all the work I've put into it then that's a problem.